scout/utils/ensembl_biomart_clients.py

```diff
@@ -29,15 +29,6 @@ def stream_get(self, url: str) -> Iterator:
     def stream_resource(self, interval_type: str) -> Iterator[str]:
         """Use schug web to fetch genes, transcripts or exons from a remote Ensembl biomart in the right genome build and save them to file."""
 
-        def yield_resource_lines(iterable) -> str:
```

Comment:
I remember introducing this code in order to remove the last line of the streamed Ensembl files, which was the following:
`[success]`

In the files downloaded using the [latest schug](https://github.com/Clinical-Genomics/schug/pull/99) - deployed now on stage - this line is missing. The last line of the downloaded file is instead the one relative to MT-TP. By removing these lines we make sure to parse the entire file, including all genes

Reply:
This change goes together with [this recent change in schug](https://github.com/Clinical-Genomics/schug/pull/87/files)